web/client/themes/default/less/common.less

```diff
@@ -1,3 +1,7 @@
+#container {
+    overflow: hidden;
```

Comment:
Too generic. This is very dangerous. Every element with id ='container' will not have scrolling anymore. I suggest you to use a less generic css selector and use the Portal element. For instance a class called fill-portal that solves this issue
some hints: 
look at :
 - z-index
 - pointer-events css class (See dashboard)